packages/odyssey-react/src/components/Form/Form.module.scss

```diff
@@ -26,7 +26,7 @@
 .header,
 .error,
 .main {
-  margin-block-end: $spacing-m;
+  margin-block-end: var(--SpaceMarginBlockEndHeaderErrorMain);
```

Comment:
lol yea when a variable is shared across multiple variants/states it gets pretty wordy - this is another challenging use case we should discuss further! Its also interesting to consider that our private implementation details (class names) now have a tendency to leak into our public API. Perhaps `Main` has meaning to an outside developer, perhaps it doesn't! When we document theme variables having a component anatomy diagram that calls out parts with the same vocabulary will help us tremendously. I have run into these issues as well cc @okta/design-system

Reply:
Could we, say, redefine the styling 3 times? Once for each (`header`, `error` and `main`) and have different variables as well?

Reply:
That's not unreasonable. We intend for these to have the same value in our theme, but I'm not sure they *need* to be uniform.

I wish we had a good way to mark these design questions for a later audit. @josesolano-okta, any thoughts?